billy/web/api/handlers.py

```diff
@@ -389,8 +390,8 @@ def read(self, request):
             resp.write(': Need lat and long parameters')
             return resp
 
-        url = "%sboundaries/?contains=%s,%s&sets=%s&limit=0" % (
-            self.base_url, latitude, longitude, settings.BOUNDARY_SERVICE_SETS
+        url = "%sdivisions/?fields=id&lat=%s&lon=%s&apikey=%s" % (
+            self.base_url, latitude, longitude, settings.API_KEY
         )
```

Comment:
proxy calls to get matching ocd ids, rather then geomatries to reduce external complexities